apps/server/src/tasks/tasks.service.ts

```diff
@@ -36,6 +40,8 @@ export class TasksService {
       repeat: repeat,
     }).save();
 
+    // console.log(chunkInfo);
```

Comment:
delete

Reply:
Was left on purpose, to not forget the place of chunkInfos xD